shared/src/business/entities/cases/Case.js

```diff
@@ -1549,6 +1704,14 @@ Case.prototype.setAsSealed = function() {
   this.isSealed = true;
   return this;
 };
+/**
+ * generates the case confirmation pdf file name
+ *
+ * @returns {string} this case confirmation pdf file name
+ */
+Case.prototype.getCaseConfirmationGeneratedPdfFileName = function() {
+  return `case-${this.docketNumber}-confirmation.pdf`;
```

Comment:
I don't know enough about docketNumber typing, but am guessing this is auto-casted into string types here. 
Are there any situations where docketNumber might be invalid or otherwise not cast correctly into a string type?

Reply:
From the test case below, it looks like actually a string, but naming it as docketNumber is also confusing if it's not a number.